From 54b66bfe405d8b70c703684e60bedbad5c7d4213 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Sat, 14 Jan 2017 21:35:53 +0200
Subject: [PATCH] tests/ts_calibrate: add missing headers

The open(2) system call requires sys/types.h and sys/stat.h. Fix the following
build failure under uClibc-ng:

ts_calibrate.c: In function ‘main’:
ts_calibrate.c:248:19: error: ‘S_IRUSR’ undeclared (first use in this function)
                   S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
                   ^

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: https://github.com/kergoth/tslib/pull/59
 tests/ts_calibrate.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/ts_calibrate.c b/tests/ts_calibrate.c
index c6652e479d4b..9a98d1f86205 100644
--- a/tests/ts_calibrate.c
+++ b/tests/ts_calibrate.c
@@ -17,6 +17,8 @@
 #include <string.h>
 #include <unistd.h>
 #include <sys/time.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
 #ifdef __ANDROID__
 #include <fcntl.h>
-- 
2.11.0

